packages/database/src/migrations/20210503082425-AddLaEocMalariaCasesBarChart-modifies-data.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const reports = [
+  {
+    id: 'Laos_EOC_Malaria_Cases_By_Week_District',
+    entityType: 'district',
+    groupCode: 'LAOS_EOC_Malaria_District',
+  },
+  {
+    id: 'Laos_EOC_Malaria_Cases_By_Week_Sub_District',
+    entityType: 'sub_district',
+    groupCode: 'LAOS_EOC_Malaria_Sub_District',
+  },
+];
+
+const buildReportConfig = (id, entityType) => ({
+  id,
+  dataBuilder: 'countEventsPerPeriod',
+  dataBuilderConfig: {
+    programCode: ['Malaria_Case_Reporting'],
+    dataElement: 'Total_Positive_Malaria_Cases',
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+      aggregationEntityType: entityType,
```

Comment:
Am I correct to understand that the only difference between these dashboards is the `aggregationEntityType`? While I wouldn't necessarily push for this right now, we actually have plans to introduce a `requested` `aggregationEntityType` which would change this entity type based on the requested entity! Would be ideal in this scenario.

I've introduced the functionality on the PSSS Phase 2 branch, but that code is a little ways off merging, perhaps lets just add a comment in this migration mentioning we should alter these dashboards once that ticket is merged. Ticket in question: https://github.com/beyondessential/tupaia-backlog/issues/2697

Reply:
Yes! I've been meaning to ask about that, we have a bunch that differ only by `aggregationEntityType` will be great to empty out some of those nigh duplicate reports!